Installing-Oppia-using-Docker.md

```diff
@@ -105,16 +106,22 @@ Download and install the latest version of Docker Desktop from the [official Doc
    This command launches the Oppia development server in offline mode, and you can continue to perform your tasks as usual.
    > NOTE: Ensure that you have already built Oppia for the first time before running this command. If not, run `make build` first (with internet connection), which downloads all the necessary third-party libraries, python dependencies, and the other services required for the Oppia development server.
 
-5. **Stop the Local Development Server**: To stop the local development server, execute the following command:
+6. **Stop the Local Development Server**: To stop the local development server, execute the following command:
 
    ```
    make stop
    ```
 
    This command stops the Oppia development server and shuts down the Docker containers running the Oppia development server.
 
+
+**Note for Developers: Allocating Resources to Docker Desktop**
+For systems with 8GB RAM:
+- Allocate at least 6GB RAM to Docker Desktop from the settings (Resources tab) in Docker Desktop.
+- Adjust CPU and swap memory according to your system specifications from the same settings panel.
+
 --------------------
-That's it! You have successfully installed Oppia using Docker, and ready-to-go for your contributions at Oppia.
+##### That's it! You have successfully installed Oppia using Docker, and ready-to-go for your contributions at Oppia.
```

Comment:
This isn't really a section header, so please don't use `#####`. If you want to emphasize this text, you can bold it

Reply:
originally if I dont use `#####`, then it is showed as header, please check [here](https://github.com/oppia/oppia/wiki/Installing-Oppia-Docker#thats-it-you-have-successfully-installed-oppia-using-docker-and-ready-to-go-for-your-contributions-at-oppia), so when I used these `#####` then it is normal bold text, so I used it.

Reply:
It's still a header, the level 5 header just looks like normal bold text. It's looking like a header because underlining text is another way to make a header in markdown. You can avoid this by putting blank lines around your horizontal lines like this:

```md
--------------------

##### That's it! You have successfully installed Oppia using Docker, and ready-to-go for your contributions at Oppia.

--------------------
```

Reply:
@shivkant5 Why not just bold the text? (E.g. put `**` around "That's it!" ... bolding just that might be easier to read than bolding the whole thing anyway.)